src/modules/universal-discovery/universal.discovery.module.js

```diff
@@ -820,14 +820,14 @@ export default class UniversalDiscoveryModule extends Component {
             attrs: {
                 languages,
                 contentTypes,
+                allowedContentTypes,
                 onLanguageSelected: this.onLanguageSelected,
                 onContentTypeSelected: this.onContentTypeSelected,
                 contentTypesMap: this.state.contentTypesMap,
                 forcedLanguage: cotfForcedLanguage,
                 preselectedLanguage: cotfPreselectedLanguage,
                 allowedLanguages: cotfAllowedLanguages,
                 preselectedContentType: cotfPreselectedContentType,
-                allowedContentTypes: cotfAllowedContentTypes,
```

Comment:
This is the configuration for content on the fly (create tab) and here should be `cotfAllowedContentTypes` passed. It was ok.